src/category_theory/limits/limits.lean

```diff
@@ -104,15 +110,48 @@ lemma hom_ext (h : is_limit t) {W : C} {f f' : W ⟶ t.X}
 by rw [h.hom_lift f, h.hom_lift f']; congr; exact funext w
 
 /--
-Given two functors which have equivalent categories of cones, we can transport a limiting cone across
-the equivalence.
+Given a right adjoint functor between categories of cones,
+the image of a limit cone is a limit cone.
 -/
-def of_cone_equiv {D : Type u'} [category.{v} D] {G : K ⥤ D}
+def of_right_adjoint {D : Type u'} [category.{v} D] {G : K ⥤ D}
   (h : cone G ⥤ cone F) [is_right_adjoint h] {c : cone G} (t : is_limit c) :
   is_limit (h.obj c) :=
 mk_cone_morphism
   (λ s, (adjunction.of_right_adjoint h).hom_equiv s c (t.lift_cone_morphism _))
-  (λ s m, (adjunction.eq_hom_equiv_apply _ _ _).2 t.uniq_cone_morphism )
+  (λ s m, (adjunction.eq_hom_equiv_apply _ _ _).2 t.uniq_cone_morphism)
+
+/--
+Given two functors which have equivalent categories of cones, we can transport a limiting cone across
+the equivalence.
+-/
+def of_cone_equiv {D : Type u'} [category.{v} D] {G : K ⥤ D}
+  (h : cone G ≌ cone F) {c : cone G} :
+  is_limit (h.functor.obj c) ≃ is_limit c :=
+{ to_fun := λ P, of_iso_limit (of_right_adjoint h.inverse P) (h.unit_iso.symm.app c),
+  inv_fun := of_right_adjoint h.functor,
+  left_inv := by tidy,
+  right_inv := by tidy, }
+
+/--
+A cone postcomposed with a natural isomorphism is a limit cone if and only if the original cone is.
+-/
+def postcompose_hom_equiv {F G : J ⥤ C} (α : F ≅ G) (c : cone F) :
+  is_limit ((cones.postcompose α.hom).obj c) ≃ is_limit c :=
+begin
+  change is_limit ((cones.postcompose_equivalence α).functor.obj c) ≃ _,
+  apply is_limit.of_cone_equiv,
+end
+
+/--
+A cone postcomposed with the inverse of a natural isomorphism is a limit cone if and only if
+the original cone is.
+-/
+def postcompose_inv_equiv {F G : J ⥤ C} (α : F ≅ G) (c : cone G) :
+  is_limit ((cones.postcompose α.inv).obj c) ≃ is_limit c :=
+begin
```

Comment:
This might not make a difference, but could we just prove this by saying `postcompose_hom_equiv a.symm c`, instead of appealing to `is_limit.of_cone_equiv`?

Reply:
Oops, yes, thanks!